ee/tabby-ui/app/(dashboard)/system/components/worker-card.tsx

```diff
@@ -200,3 +224,55 @@ function ModelIcon({ type }: { type: string }) {
     )
   }
 }
+
+function HealthInfoView({
+  backend,
+  className
+}: {
+  backend: ModelHealthBackend
+  errorMessage?: string
+  className?: string
+}) {
+  const [{ data, fetching, stale, error }, reexecuteQuery] = useQuery({
+    query: testModelConnectionQuery,
+    variables: {
+      backend
+    }
+  })
+
+  const connected = !isNil(data?.testModelConnection?.latencyMs)
+
+  if (fetching || stale) {
+    return (
+      <div className={cn(className)}>
+        <IconSpinner />
+      </div>
+    )
+  }
+
+  return (
+    <div className={cn('flex items-center gap-3', className)}>
+      {connected ? (
+        <p className="flex items-center gap-0.5 font-semibold text-green-600 dark:text-green-500">
+          {/* <IconCheck className='h-3.5 w-3.5' /> */}
+          Available
+        </p>
+      ) : (
+        <div className="flex items-center gap-0.5 font-semibold text-red-600 dark:text-red-500">
+          <IconCircleAlert className="h-3.5 w-3.5" />
+          Unavailable
+        </div>
+      )}
+
+      <Button
+        className="h-5 w-5"
+        size="icon"
+        variant="ghost"
+        disabled={fetching}
+        onClick={reexecuteQuery}
+      >
+        <IconRefresh className="h-3.5 w-3.5" />
```

Comment:
```suggestion
        <IconRefresh className="h-3.5 w-3.5" />
```

Use radix reload: https://www.radix-ui.com/icons